tests/testthat/test-interface.R

```diff
@@ -39,6 +39,37 @@ test_that("can submit a task via a driver", {
     id %in% names(hipercow_root(path_here)$cache$task_status_terminal))
 })
 
+test_that("Can submit a task with hold_until special keywords", {
+  elsewhere_register()
+  path_here <- withr::local_tempdir()
+  path_there <- withr::local_tempdir()
+
+  init_quietly(path_here)
+  init_quietly(path_there)
+
+  res <- hipercow_resources(hold_until = "midnight")
+  suppressMessages(
+    hipercow_configure("elsewhere", path = path_there, root = path_here))
+
+  mock_special_time <- mockery::mock("midnight2")
+  mockery::stub(task_submit, "special_time", mock_special_time)
+
+  expect_message(
+    id <- withr::with_dir(path_here, task_create_explicit(quote(getwd()),
+                        resources = res)),
+    "Submitted task '[[:xdigit:]]{32}' using 'elsewhere'")
+
+  # TODO HERE - Something is wrong with my mockery - mock_special_time
```

Comment:
I'll talk you through this in person, it's an unfortunate consequence of how mockery works.

Usually some contortions are required here to make good tests